src/app/components/fileattachments/index.html

```diff
@@ -1,25 +1,213 @@
+		</sky-docs-visual-example>
+    
+    <h3>States</h3>
+    <div class="sky-margin-stacked-content-section">
+    <sky-demo-page-component-states>
+      <sky-demo-page-component-state stateName="Base" imagePath="~/assets/img/guidelines/single-file-attach/state-base.png"></sky-demo-page-component-state>
+      <sky-demo-page-component-state stateName="Attached, image" imagePath="~/assets/img/guidelines/single-file-attach/state-attached-image.png"></sky-demo-page-component-state>
+      <sky-demo-page-component-state stateName="Attached, non-image" imagePath="~/assets/img/guidelines/single-file-attach/state-attached-non-image.png"></sky-demo-page-component-state>
+      <sky-demo-page-component-state stateName="Error" imagePath="~/assets/img/guidelines/single-file-attach/state-error.png"></sky-demo-page-component-state>
+    </sky-demo-page-component-states>
+    </div>
+	</sky-demo-page-content>
+  
+  <sky-demo-page-content sectionHeading="Related information">
+		<h3>Guidelines</h3>
+			<ul>
+				<li><a routerLink="/design/guidelines/form-design">Form design</a></li>
+			</ul>
+	</sky-demo-page-content>
+    
+  </sky-tab>
+
+  <sky-tab
+		tabHeading="Design - file attachment">
+    <sky-demo-page-content sectionHeading="Usage">
+    <h3 class="sky-text-success">Use when</h3>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>Use file attachment when the user needs to upload multiple files or attach a file from a URL rather than the local device.</p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <div>
+            <img src="~/assets/img/guidelines/file-attach/use-multi.png" />
+            <div class="caption caption-success sky-text-success">Do use file attachment to upload more than one file.</div>
+          </div>
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>Use file attachment when the user needs to add metadata to the uploaded file.</p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <div>
+            <img src="~/assets/img/guidelines/file-attach/use-metadata.png" />
+            <div class="caption caption-success sky-text-success">Do use file attachment to add metadata about the uploaded file.</div>
+          </div>
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+
+    <h3 class="sky-text-danger">Don't use when</h3>
+      <p class="sky-text-block">
+        Don't use file attachment if the user needs to upload one local file. Use single file attachment instead.
+      </p>
+  </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Anatomy">
+      <sky-docs-anatomy imagePath="~/assets/img/guidelines/file-attach/anatomy.png">
+        <sky-docs-anatomy-item anatomyNumber="1">Browse or drop local file</sky-docs-anatomy-item>
```

Comment:
This item and #6 both feel like they are missing a noun. All the other items in the list are nouns that correspond to UI items. But #1 and #6 are the actions that the UI items enable. What do you think of changing these to something like `Upload file option` and `Attach URL option`?

Reply:
updated